lib/src/main/java/io/ably/lib/objects/LiveObjects.java

```diff
@@ -0,0 +1,8 @@
+package io.ably.lib.objects;
+
+import io.ably.lib.types.Callback;
+
+public interface LiveObjects {
```

Comment:
Why can't we call this singular `LiveObject` instead? 
Like it's holding only one parent `rootObject`

Reply:
We don't have methods suggesting any plural values as well

Reply:
Don't have any opinion here, both singular and plural are good for me, this PR just a reference for the plugin setup in gradle

Reply:
Guess it represents type of returned `getObjects` method